packages/gradient_boosting_model/gradient_boosting_model/datasets/houseprice.csv

```diff
@@ -0,0 +1,1461 @@
+Id,MSSubClass,MSZoning,LotFrontage,LotArea,Street,Alley,LotShape,LandContour,Utilities,LotConfig,LandSlope,Neighborhood,Condition1,Condition2,BldgType,HouseStyle,OverallQual,OverallCond,YearBuilt,YearRemodAdd,RoofStyle,RoofMatl,Exterior1st,Exterior2nd,MasVnrType,MasVnrArea,ExterQual,ExterCond,Foundation,BsmtQual,BsmtCond,BsmtExposure,BsmtFinType1,BsmtFinSF1,BsmtFinType2,BsmtFinSF2,BsmtUnfSF,TotalBsmtSF,Heating,HeatingQC,CentralAir,Electrical,1stFlrSF,2ndFlrSF,LowQualFinSF,GrLivArea,BsmtFullBath,BsmtHalfBath,FullBath,HalfBath,BedroomAbvGr,KitchenAbvGr,KitchenQual,TotRmsAbvGrd,Functional,Fireplaces,FireplaceQu,GarageType,GarageYrBlt,GarageFinish,GarageCars,GarageArea,GarageQual,GarageCond,PavedDrive,WoodDeckSF,OpenPorchSF,EnclosedPorch,3SsnPorch,ScreenPorch,PoolArea,PoolQC,Fence,MiscFeature,MiscVal,MoSold,YrSold,SaleType,SaleCondition,SalePrice
```

Comment:
Hi Chris. We are not allowed to re-host the data set. That is a violation of copyright. We do have permission from Prof DeCock to use it, on the condition we give him a copy of the product we built using his data set. He has given us a link to the original data. We are however using the data set from Kaggle. We do have permission from Kaggle to direct students to download the data from their site. But re-hosting is not allowed. Can we re do this so that they download the data set themselves, and therefore can accept the T&C from Kaggle when they do so? Or alternatively, use the Kaggle API for downloading. We need to remove the dataset from the commit history as well. Sorry about this. Cheers. Sole @ChristopherGS

Reply:
Done